src/components/ItemDetail/use/return.ts

```diff
@@ -0,0 +1,76 @@
+import { ref, Ref, ComputedRef, computed } from 'vue'
+import apis, { ItemSummary, Owner, LogType } from '/@/lib/apis'
+import { OwnerWithCount } from './owners'
+import { stringifyDate } from '/@/lib/date'
+
+const useReturn = (props: {
+  item: ItemSummary
+}): {
+  details: ComputedRef<OwnerWithCount[]>
+  selectedOwnerName: Ref<string>
+  count: Ref<number>
+  owner: ComputedRef<Owner | undefined>
+  returnItem: () => Promise<void>
+} => {
+  // TODO: storeにmeをおいたらIDにちゃんとしたやつを入れる(10は@ryoha のID)
+  const details = computed(
+    () =>
+      props.item.rentalUsers
+        .filter(v => v.userId === 10 && v.count !== 0)
+        ?.map(v => ({
+          userName: v.owner.name,
+          count: v.count * -1
+        })) ?? []
```

Comment:
```suggestion
        .map(v => ({
          userName: v.owner.name,
          count: v.count * -1
        }))
```
filterは常に配列返すはずだから`?`いらないと思う

Reply:
findの名残だったわ